Project.toml

```diff
@@ -16,7 +16,7 @@ TimeZones = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 
 [compat]
 Downloads = "1"
-OpenAPI = "0.1"
+OpenAPI = "0.1,0.2"
```

Comment:
Since Kuber.jl is a registered package, we probably need to also up the patch version of the package for a new registration?